modules/twinklewarn.js

```diff
@@ -221,27 +227,29 @@ Twinkle.warn.messages = {
 				label: "Ownership of articles",
 				summary: "General note: Ownership of articles"
 			},
+			"uw-plotsum1": {
+				label: "Inappropriate expansion of plot summaries",
+				summary: "General note: Inappropriate expansion of plot summaries"
+			},
+			"uw-taxonomy1": {
+				label: "Disrupting the taxonomy templates",
+				summary: "General note: Disrupting the taxonomy templates"
+			},
```

Comment:
This is certainly a rarity (broadly speaking), and doesn't need to be cluttering the Twinkle interface.